components/notifier.rst

```diff
@@ -26,3 +26,44 @@ Usage
 
 	We're still working on the docs of this component. Check this page again
 	in a few days.
+
+Slack Actions Block for Slack Message
+-------------------------------------
+
+With a Slack Message you can add some interactive options called `Block elements`_::
+
+    use Symfony\Component\Notifier\Bridge\Slack\Block\SlackActionsBlock;
+    use Symfony\Component\Notifier\Bridge\Slack\SlackOptions;
+    use Symfony\Component\Notifier\Bridge\Slack\SlackTransport;
+    use Symfony\Component\Notifier\Chatter;
+    use Symfony\Component\Notifier\Message\ChatMessage;
+
+    // Initialize a chatter with Slack Transport
+    $chatter = new Chatter(new SlackTransport('token'));
+
+    // Create a message
+    $chatMessage = (new ChatMessage())
```

Comment:
Parse the subject string at the constructor.
`$chatMessage = new ChatMessage('Contribute to Symfony');`